assets/keymaps/default-linux.json

```diff
@@ -587,7 +587,7 @@
       "f2": "project_panel::Rename",
       "enter": "project_panel::Rename",
       "shift-delete": ["project_panel::Delete", { "skip_prompt": false }],
-      "delete": ["project_panel::Trash", { "skip_prompt": true }],
+      "delete": ["project_panel::Trash", { "skip_prompt": false }],
```

Comment:
To note, this alone will remove all keybindings from mac menu entries.

Reply:
> To note, this alone will remove all keybindings from mac menu entries.

I don't understand, please say more.

1. Isn't this `-linux.json`?
2. How does adding a single binding to scoped to ProjectPanel remove all keybindings for mac menu entries?

Reply:
Sorry, I've actually missed that this fixes the issue with Linux keybindings, and my comment is also wrong as was about mac, please disregard it entirely.

1. Yes, sorry.
2. It's not adding, but altering the keybinding's value what happens in the PR. If that gets out of sync with the value, defined in the menu, the binding keys won't be displayed in the UI.

In this case they actually start to match, so all good:
https://github.com/zed-industries/zed/blob/1e0e1c7c3199c412b49cbed56c7ad4087fc5842e/crates/project_panel/src/project_panel.rs#L504

Reply:
Thank you. This is helpful to know that the menu items and keybinds could be out of sync in the actions they perform.